src/app/lists/EditModal.tsx

```diff
@@ -115,12 +173,12 @@ const EditModal: React.FC<Props> = ({ data, onClose }) => {
                             <h3>{label}</h3>
                             <DataWithTooltipDiv>
                                 <TextInput
-                                    defaultValue={toSubmit[quantityKey]?.toString() ?? ""}
+                                    defaultValue={toSubmit ? toSubmit[quantityKey]?.toString() : ""}
```

Comment:
Are you aware that this expression is either string or undefined since `toSubmit[quantityKey]?.toString()` can be undefined due to optional chaining? I'm guessing this makes no diff in the behaviour but just wanted to make sure you're aware